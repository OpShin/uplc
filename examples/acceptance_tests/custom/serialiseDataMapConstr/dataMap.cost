{
  "cpu": 130809924,
  "mem": 1060
}
