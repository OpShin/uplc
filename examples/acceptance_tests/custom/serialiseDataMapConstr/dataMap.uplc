(program 1.0.0 [(builtin serialiseData) (con    data    (Constr 0       [ Constr 0           [ List               [ Constr 0                   [ Constr 0                       [ Constr 0                           [ B #fda74de33747f6a025e0ca9865c7fad449d98b593dca6e065d5ff89afdbf6d9b ]                       , I 0 ]                   , Constr 0                       [ Constr 0                           [ Constr 1                               [ B #6a7b88dbd9b42eb01dd9be9fd7d49801c9dc0b32552ff823193cd163 ]                           , Constr 1 [] ]                       , Map [(B #, Map [(B #, I 3000000)])]                       , Constr 2 [Constr 0 []]                       , Constr 1 [] ] ] ]           , List []           , List               [ Constr 0                   [ Constr 0                       [ Constr 0                           [ B #61299458bd6d3011669ac533b520ab07b94d7428903f61714babb582 ]                       , Constr 1 [] ]                   , Map [(B #, Map [(B #, I 2815931)])]                   , Constr 0 []                   , Constr 1 [] ] ]           , Map [(B #, Map [(B #, I 184069)])]           , Map [(B #, Map [(B #, I 0)])]           , List []           , Map []           , Constr 0               [ Constr 0 [Constr 1 [I 1745484590000], Constr 1 []]               , Constr 0 [Constr 1 [I 1745485254000], Constr 0 []] ]           , List []           , Map               [ ( Constr 1                     [ Constr 0                         [ Constr 0                             [ B #fda74de33747f6a025e0ca9865c7fad449d98b593dca6e065d5ff89afdbf6d9b ]                         , I 0 ] ]               , I 0 ) ]           , Map []           , Constr 0               [ B #5441326c60e5526246140612eb8a6d1f72e011cda2e0f7c3b0e65225740774d4 ] ]       , Constr 1           [ Constr 0               [ Constr 0                   [ B #fda74de33747f6a025e0ca9865c7fad449d98b593dca6e065d5ff89afdbf6d9b ]               , I 0 ] ] ])  )])
