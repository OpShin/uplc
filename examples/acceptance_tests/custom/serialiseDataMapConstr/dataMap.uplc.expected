(program 1.0.0 (con bytestring #d8799fd8799f81d8799fd8799fd8799f5820fda74de33747f6a025e0ca9865c7fad449d98b593dca6e065d5ff89afdbf6d9bff00ffd8799fd8799fd87a9f581c6a7b88dbd9b42eb01dd9be9fd7d49801c9dc0b32552ff823193cd163ffd87a80ffa140a1401a002dc6c0d87b9fd87980ffd87a80ffff8081d8799fd8799fd8799f581c61299458bd6d3011669ac533b520ab07b94d7428903f61714babb582ffd87a80ffa140a1401a002af7bbd87980d87a80ffa140a1401a0002cf05a140a1400080a0d8799fd8799fd87a9f1b0000019666fd2bb0ffd87a80ffd8799fd87a9f1b0000019667074d70ffd87980ffff80a1d87a9fd8799fd8799f5820fda74de33747f6a025e0ca9865c7fad449d98b593dca6e065d5ff89afdbf6d9bff00ffff00a0d8799f58205441326c60e5526246140612eb8a6d1f72e011cda2e0f7c3b0e65225740774d4ffffd87a9fd8799fd8799f5820fda74de33747f6a025e0ca9865c7fad449d98b593dca6e065d5ff89afdbf6d9bff00ffffff))